embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/SpanAttributeMapper.kt

```diff
@@ -0,0 +1,8 @@
+package io.embrace.android.embracesdk.arch
+
+/**
+ * Converts an object of type T to a [SpanAttributeData]
+ */
+internal fun interface SpanAttributeMapper<T> {
```

Comment:
These two mappers are not being used. Are they gonna be used in another PR?

Reply:
Yep - I've got an example of how they work in #406.